pkgs/top-level/linux-kernels.nix

```diff
@@ -241,6 +241,12 @@ in {
     linux_xanmod_stable = xanmodKernels.main;
     linux_xanmod_latest = xanmodKernels.main;
 
+    sunlightKernels = callPackage ../os-specific/linux/kernel/sunlight-kernels.nix {
+      kernelPatches = [];
```

Comment:
Do you incorporate our patches?

Reply:
No, i not have patch files.

You suggest to remove this?

Reply:
In that case you should probably put the kernel patches in that we include in our other kernels of this version.

Reply:
I mean the kernels we put into every kernel in Nixpkgs. Please take a look at how it's done for the zen kernels or xanmod.

Reply:
I checked how it looks in xanmod and zen, in the patch area(kernelPatches), I made an update for this aspect.